### DISCUSSION TITLE
Build rpm

### DISCUSSION THREAD

**Discussion by cvas71** [okbob/plpgsql_check#110] [General]
Good day.
In order for me to be able to compile the rpm, I had to slightly change postgresql14-plpgsql_check.spec.
Changed:
USE_PGXS=1 make %{?_smp_mflags} -> make USE_PGXS=1 PG_CONFIG=%{pginstdir}/bin/pg_config %{?_smp_mflags}
make USE_PGXS=1 DESTDIR=%{buildroot} install -> make install DESTDIR=%{buildroot} PG_CONFIG=%{pginstdir}/bin/pg_config %{?_smp_mflags}
%{pginstdir}/share/extension/plpgsql_check--2.0.sql -> %{pginstdir}/share/extension/plpgsql_check--2.1.sql
Added:
%{pginstdir}/lib/bitcode/*.bc
%{pginstdir}/lib/bitcode/plpgsql_check/src/*.bc
%{pginstdir}/share/extension/*.control
Changed
* Fri Dec 23 2018 - Pavel STEHULE <pavel.stehule@gmail.com> 1.4.2-1 -> * Sun Dec 23 2018 - Pavel STEHULE <pavel.stehule@gmail.com> 1.4.2-1
* Fri Now 11 2016 - Pavel STEHULE <pavel.stehule@gmail.com> 1.2.0-1 -> * Fri Nov 11 2016 - Pavel STEHULE <pavel.stehule@gmail.com> 1.2.0-

**Comment by okbob** [comment]
please, can you prepare pull request?

**Comment by cvas71** [comment]
[postgresql14-plpgsql_check.zip](https://github.com/okbob/plpgsql_check/files/8684363/postgresql14-plpgsql_check.zip)

**Reply by cvas71** [reply]
I haven't worked much with Git, so I don't know much about all of its features.

**Comment by okbob** [comment]
Hi

should be fixed now in master branch

Thank you for fixed spec file

Regards

Pavel

pá 13. 5. 2022 v 7:48 odesílatel Constantine Grischenko <
***@***.***> napsal:
